.env.local.text

```diff
@@ -0,0 +1,8 @@
+NEXT_PUBLIC_SUPABASE_URL=https://mpwymicewrcflpddvdlo.supabase.co
+NEXT_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoiYW5vbiIsImlhdCI6MTYzMDI4MTIxMCwiZXhwIjoxOTQ1ODU3MjEwfQ.iwpk2cKNcavD3YGNHhG3KfCpJhAkXEbvjOjkkRwvrC0
+SENDGRID_API_KEY=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
+
+NEXT_PUBLIC_SUPABASE_URL=http://localhost:54321
+NEXT_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24ifQ.625_WdcF3KHqz5amU0x2X5WWHP-OEs_4qj0ssLNHzTs
+SENDGRID_API_KEY=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
```

Comment:
I think we need to add this to .gitignore 🤔 . Did you get an email from sendgrid or anything about exposing keys? Or is this the old key?